test-app/src/components/debugger.tsx

```diff
@@ -51,6 +55,18 @@ export const Debugger = () => {
     setTxType(type);
   };
 
+  // If need to add more test tokens: STW7PFH79HW1C9Z0SXBP5PTPHKZZ58KK9WP1MZZA
+  const handleSponsoredTransactionBroadcast = async (tx: StacksTransaction) => {
+    const sponsorOptions = {
+      transaction: tx,
+      sponsorPrivateKey: 'b8c6aaef4b6de5e62648a59fff13e389856dff5e58163e676c2cfdf9dd5dd11101',
+      fee: new BN(388),
+    };
+
+    const sponsoredTx = await sponsorTransaction(sponsorOptions);
+    broadcastTransaction(sponsoredTx, network);
```

Comment:
```suggestion
    return broadcastTransaction(sponsoredTx, network);
```
?

Reply:
This did work but seems like it should be there, thx.